src/banana/data/db.py

```diff
@@ -21,10 +21,9 @@
 class MyBase:
     uid = Column(Integer, primary_key=True, unique=True)
     hash = Column(String(64), unique=True)
-    # TODO: should we use `func.utcnow`?
-    # https://stackoverflow.com/a/33532154/8653979
```

Comment:
Regarding this the answer is simply 'no'. We could provide the function that way, but we should implement ourselves in any case (since it's not available from databases by default, not all, most likely not SQLite).

Reply:
In sqlite it is, and it's the only one we care about